homework_1/01_FizzBuzz/fizzbuzz/fizzBuzz.go

```diff
@@ -0,0 +1,35 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func GetFizzBuzzSolution(start int, end int) ([]string, error) {
+	if start > end {
+		return nil, errors.New("Start is greater than end.")
+	}
+
+	if start < 1 {
+		return nil, errors.New("Start is lower than 1.")
+	}
+
+	var solution []string
+
+	for ;start <= end; start++ {
```

Comment:
You're using `start` variable here to iterate through all the numbers. This is kind-of misleading due to its name.
It might _seem_ like a good idea to reuse a variable so you don't need to define a new one but making code less readable makes it worse overall.

If you use a new variable here to iterate through here, like this `for n := start; n <= end; n++` for example:
* you'll make the code more readable and digestible for others
* the compiler will optimise away that variable anyway